{{cookiecutter.project_slug}}/config/settings/production.py

```diff
@@ -48,6 +48,20 @@
     },
 }
 
+
+REDIS_URL = env("REDIS_TLS_URL", default=env("REDIS_URL"))
+REDIS_SSL = env.bool("REDIS_SSL", default=False)
+
+CERT_NONE = 0
+
+if REDIS_SSL:
+    CELERY_REDIS_BACKEND_USE_SSL = {"ssl_cert_reqs": CERT_NONE}
+    CELERY_BROKER_USE_SSL = {"ssl_cert_reqs": CERT_NONE}
+    CACHES["default"]["OPTIONS"]["CONNECTION_POOL_CLASS"] = "redis.connection.SSLConnection"
+    CACHES["default"]["OPTIONS"]["SSL_CERT_REQS"] = CERT_NONE
```

Comment:
Why do you need that? The cache seems to work fine on my end with the new secure redis connection string... What am I missing?

Reply:
To facilitate testing in a development environment, we have added a setting to disable SSL certificate verification.
However, as you pointed out, if it works well with the default settings,
I think it would be cleaner to remove unnecessary settings and handle SSL connections with only the 'rediss://' URL.